app/javascript/components/background.js

```diff
@@ -0,0 +1,12 @@
+const changeColor = () => {
+
+   let hue = 0;
+    function changeBg() {
+      if (hue >= 360) hue = 0;
+      document.body.style.backgroundColor = "hsl(" + hue + ",80%,50%)";
```

Comment:
change to banner.style.backgroundColor = "hsl(" + hue + ",80%,50%)";

Reply:
ok